.travis.yml

```diff
@@ -31,6 +31,10 @@ jobs:
       dist: xenial
       env:
         - TEST_FLAKE8="true"
+    - python: 3.8
+      dist: xenial
+      env:
```

Comment:
Any reason not to merge this with the previous build? Travis limits the number of concurrent jobs, so if two jobs can be merged into one that is preferable.

Reply:
Using separate jobs reduces the total time taken for this stage of the build. Running jobs concurrently is generally a good thing although it does include a small overhead for job startup.